### PR TITLE
Persit option

### LINKED ISSUE #30: Configure log record in the PersistEngine
PersitEngineConfig : 
logOnSucces : true/false
logOnError : true/false
logLevelOnSucess : TRACE/DEBUG/INFO/WARN/ERROR
logLevelOnError : TRACE/DEBUG/INFO/WARN/ERROR

### PR DESCRIPTION
**Pull request by PierreLeresteux**
Proposition to parametrize persist_log

Close #30

### CODE REVIEW

File: core/src/main/kotlin/io/saagie/updatarium/dsl/ChansetSet.kt [hunk 1]
```diff
@@ -53,11 +53,13 @@ data class ChangeSet(
...
                 InMemoryAppenderManager.stopRecord()
-                persistEngine.unlock(this, Status.OK,InMemoryAppenderAccess.getEvents())
+                persistEngine.unlock(this, Status.OK, InMemoryAppenderAccess
+                    .getEvents(persistConfig = persistEngine.configuration,success = true))
                 logger.info { "$id marked as ${Status.OK}" }
             } catch (e: Exception) {
                 logger.error(e) { "Error during apply update" }
-                persistEngine.unlock(this, Status.KO, InMemoryAppenderAccess.getEvents())
+                persistEngine.unlock(this, Status.KO, InMemoryAppenderAccess
+                    .getEvents(persistConfig = persistEngine.configuration,success = false))
```
Comment (rcapraro) [inline core/src/main/kotlin/io/saagie/updatarium/dsl/ChansetSet.kt]: missing space between `persistEngine.configuration` and `success = false`

File: core/src/main/kotlin/io/saagie/updatarium/dsl/ChansetSet.kt [hunk 2]
```diff
@@ -53,11 +53,13 @@ data class ChangeSet(
                     it.execute()
                 }
                 InMemoryAppenderManager.stopRecord()
-                persistEngine.unlock(this, Status.OK,InMemoryAppenderAccess.getEvents())
+                persistEngine.unlock(this, Status.OK, InMemoryAppenderAccess
+                    .getEvents(persistConfig = persistEngine.configuration,success = true))
```
Comment (rcapraro) [inline core/src/main/kotlin/io/saagie/updatarium/dsl/ChansetSet.kt]: missing space between persistEngine.configuration and success = false

File: core/src/main/kotlin/io/saagie/updatarium/dsl/ChansetSet.kt (hunk 1 above)
Comment (PierreLeresteux) [reply core/src/main/kotlin/io/saagie/updatarium/dsl/ChansetSet.kt]: LinterMan !

Comment (PierreLeresteux) [reply core/src/main/kotlin/io/saagie/updatarium/dsl/ChansetSet.kt]: Fixed

File: core/src/main/kotlin/io/saagie/updatarium/dsl/ChansetSet.kt (hunk 2 above)
Comment (PierreLeresteux) [reply core/src/main/kotlin/io/saagie/updatarium/dsl/ChansetSet.kt]: Fixed

**Pull request merged**